lego/apps/companies/views.py

```diff
@@ -47,9 +51,7 @@
 class AdminCompanyViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
     queryset = Company.objects.all().prefetch_related("semester_statuses", "files")
     permission_handler = CompanyAdminPermissionHandler()
-    filter_backends = [filters.OrderingFilter]
-    ordering_fields = ["name", "created_at"]
-    ordering = "name"
+    filter_backends = [CompanyOrderingFilter]
```

Comment:
```suggestion
    filterset_class = AdminCompanyFilterSet
```

Not sure if I get the difference, but nice to be consistent and use the `django_filters` library